### PR TITLE
OH2-241 Update Examination default values

### PR DESCRIPTION
**Pull request by ArnaudFonzam**
See OH2-241.

I have use the values specifies in examination.properties to update this file

- add enum Bowel, Ausculation and Diurese

### CODE REVIEW

File: src/main/java/org/isf/generaldata/ExaminationParameters.java [hunk 1]
```diff
@@ -95,6 +95,9 @@ public final class ExaminationParameters extends ConfigurationProperties {
 	
 	public static int SAT_MIN;
 	private static final int DEFAULT_SAT_MIN = 50;
+	
+	public static int SAT_MAX;
+	private static final int DEFAULT_SAT_MAX = 100;
```
Comment (mwithi) [inline src/main/java/org/isf/generaldata/ExaminationParameters.java:100]: This is not needed because saturation cannot be above 100%

**Review by mwithi — CHANGES_REQUESTED** [review]
Revert change on SAT_MAX

File: src/main/java/org/isf/generaldata/ExaminationParameters.java (hunk 1 above)
Comment (ArnaudFonzam) [reply src/main/java/org/isf/generaldata/ExaminationParameters.java:100]: I use that value to check if saturation give by use is less than that Max value

Comment (mwithi) [reply src/main/java/org/isf/generaldata/ExaminationParameters.java:100]: I understand, but it is not something that can be decided by the user, it is a percentage :)
so I think you can just hardcode it or at least don't allow the user to modify it like other parameters in line 156

Comment (ArnaudFonzam) [reply src/main/java/org/isf/generaldata/ExaminationParameters.java:100]: okay can I fix value 99 to that param

Comment (mwithi) [reply src/main/java/org/isf/generaldata/ExaminationParameters.java:100]: why 99? saturation can be 100 and it could means hyperventilation...

Comment (mwithi) [reply src/main/java/org/isf/generaldata/ExaminationParameters.java:100]: you can just put hardcode 100 or if you want you can keep it and remove the line 156 further down

Comment (mwithi) [inline src/main/java/org/isf/generaldata/ExaminationParameters.java:100]:
```suggestion
	public static int SAT_MAX = 100;
```

File: src/main/java/org/isf/generaldata/ExaminationParameters.java [hunk 2]
```diff
@@ -150,6 +153,7 @@ private ExaminationParameters(String fileProperties) {
 		TEMP_STEP = myGetProperty("TEMP_STEP", DEFAULT_TEMP_STEP);
 
 		SAT_MIN = myGetProperty("SAT_MIN", DEFAULT_SAT_MIN);
+		SAT_MAX = myGetProperty("SAT_MAX", DEFAULT_SAT_MAX);
```
Comment (mwithi) [inline src/main/java/org/isf/generaldata/ExaminationParameters.java:156]:
```suggestion
```

**Review by mwithi — APPROVED** [review]

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**